src/gui/DatabaseOpenWidget.cpp

```diff
@@ -284,7 +285,11 @@ void DatabaseOpenWidget::openDatabase()
             auto keyData = databaseKey->serialize();
 #if defined(Q_CC_MSVC)
             // Store the password using Windows Hello
-            getWindowsHello()->storeKey(m_filename, keyData);
+            if (!getWindowsHello()->storeKey(m_filename, keyData)) {
+                getMainWindow()->displayTabMessage(
+                    tr("Windows Hello setup was canceled or failed. Quick unlock has not been enabled."),
+                    MessageWidget::MessageType::Warning);
```

Comment:
I've suggested an  "Enable QuickUnlock" checkbox on the main Database Unlock widget.

Reply:
That's yet another interface component there. We already have way too many.

Reply:
It's bad enough we have to show the windows hello prompt in the first place.

Reply:
> That's yet another interface component there. We already have way too many.

There was also a suggestion to hide the additional credentials behind a toggle (likely a checkbox). Not directly related to this PR, but if implemented, it would reduce the default visible controls to just the password box and 1-2 checkboxes.

Reply:
We'll investigate doing this for 2.8.0